samples/client/petstore/swift/default/PetstoreClient/Classes/Swaggers/Models/EnumArrays.swift

```diff
@@ -26,6 +26,7 @@ open class EnumArrays: JSONEncodable {
     func encodeToJSON() -> Any {
         var nillableDictionary = [String:Any?]()
         nillableDictionary["just_symbol"] = self.justSymbol?.rawValue
+        nillableDictionary["array_enum"] = self.arrayEnum?.encodeToJSON()
```

Comment:
Right, something like:

nillableDictionary["array_enum"] = array_enum?.map({$0.rawValue}).encodeToJSON()

Will add it later this week